scripts/end_to_end_demo.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
i think we should reference microk8s in the name of the script.  maybe end_to_end_microk8s_demo.sh

Reply:
After I commit this change, I'll revise the GCE|DO docs to reflect it.

Reply:
Fixed: https://github.com/deislabs/akri/pull/56/commits/a4eaae5aa06c920c8033c226bccdf739b81c0cc9

Reply:
Revised script filename:
+ Compute Engine PR #57 (Commit: https://github.com/deislabs/akri/pull/57/commits/c41ace92f688ac5c5c3d81ff8860efd7812f192f)
+ DigitalOcean PR #58 : (Commit: https://github.com/deislabs/akri/pull/58/commits/424f6e64779bacafd0b6bc87659413c45688f673)